app/models/form/sales/questions/property_local_authority_known.rb

```diff
@@ -3,7 +3,7 @@ def initialize(id, hsh, page)
     super
     @id = "la_known"
     @check_answer_label = "Local authority known"
-    @header = "Do you know the local authority of the property?"
+    @header = "Do you know the property’s local authority?"
```

Comment:
won't there be a test file for this given it's in sales?

Reply:
nothing failed so it's not covered. do we need tests to cover copy?